apps/journeys-admin/src/components/HelpScoutBeacon/HelpScoutBeacon.tsx

```diff
@@ -121,7 +121,10 @@ export function HelpScoutBeacon({
           right: { xs: 10, md: 10 },
           width: 24,
           height: 24,
-          color: matches ? 'secondary.dark' : 'background.paper'
+          color:
+            matches || userInfo.email === null
```

Comment:
Update `matches` and chance userInfo to a nullish check instead

```suggestion
            mdUp || userInfo.email == null
```